External derby tests error '/derby/build.xml:212: Compiler level must be Java 9 or later.'

External derby tests error '/derby/build.xml:212: Compiler level must be Java 9 or later.'

```
08:51:43 Buildfile: /derby/build.xml
08:51:45 
08:51:45 clean:
08:51:45 
08:51:45 cleangenerated:
08:51:45 
08:51:45 cleanstate:
08:51:45 
08:51:45 cleanreleasefiles:
08:51:45 
08:51:45 clobber:
08:51:45 
08:51:45 BUILD SUCCESSFUL
08:51:45 Total time: 1 second
08:51:45 Buildfile: /derby/build.xml
08:51:47 
08:51:47 checkCompilerLevel:
08:51:47 
08:51:47 BUILD FAILED
08:51:47 /derby/build.xml:212: Compiler level must be Java 9 or later.
08:51:47 
08:51:47 Total time: 1 second
08:51:47 Buildfile: /derby/build.xml
```

https://ci.adoptopenjdk.net/view/Test_external/job/openjdk8_j9_externaltest_x86-64_linux/345/consoleFull

--- a/thirdparty_containers/derby/dockerfile/Dockerfile
+++ b/thirdparty_containers/derby/dockerfile/Dockerfile
@@ -40,20 +40,8 @@ RUN apt-get update \
 	unzip \
 	vim \
 	tar \
-	wget
-
-# Installs Ant 1.10.2 as required by Derby 
-ENV ANT_VERSION 1.10.2 
-ENV ANT_HOME=/opt/ant
-RUN wget --no-check-certificate --no-cookies https://archive.apache.org/dist/ant/binaries/apache-ant-${ANT_VERSION}-bin.tar.gz \
-    && wget --no-check-certificate --no-cookies https://archive.apache.org/dist/ant/binaries/apache-ant-${ANT_VERSION}-bin.tar.gz.md5 \
-    && echo "$(cat apache-ant-${ANT_VERSION}-bin.tar.gz.md5) apache-ant-${ANT_VERSION}-bin.tar.gz" | md5sum -c \
-    && tar -zvxf apache-ant-${ANT_VERSION}-bin.tar.gz -C /opt/ \
-    && ln -s /opt/apache-ant-${ANT_VERSION} /opt/ant \
-    && rm -f apache-ant-${ANT_VERSION}-bin.tar.gz \
-    && rm -f apache-ant-${ANT_VERSION}-bin.tar.gz.md5
-RUN update-alternatives --install "/usr/bin/ant" "ant" "/opt/ant/bin/ant" 1 && \
-    update-alternatives --set "ant" "/opt/ant/bin/ant" 
+	wget \
+	ant
 
 VOLUME ["/java"]
 ENV JAVA_HOME=/java \

--- a/thirdparty_containers/derby/dockerfile/derby-test.sh
+++ b/thirdparty_containers/derby/dockerfile/derby-test.sh
@@ -21,7 +21,7 @@ if [ -d /java/jre/bin ];then
 	export PATH=$JAVA_BIN:$PATH
 	java -version
 elif [ -d /java/bin ]; then
-	echo "Using mounted Java9"
+	echo "Using mounted Java"
 	export JAVA_BIN=/java/bin
 	export JAVA_HOME=/java
 	export PATH=$JAVA_BIN:$PATH
@@ -37,17 +37,8 @@ else
 	export JAVA_HOME="${java_root%/bin}"
 fi
 
-TEST_SUITE=$1
-
-echo "PATH is : $PATH"
-echo "JAVA_HOME is : $JAVA_HOME"
-echo "type -p java is :"
-type -p java
-echo "java -version is: \n"
-java -version
-
 cd ${DERBY_HOME}
-ls .
+
 pwd
 
 #clean previous build artifacts 

--- a/thirdparty_containers/derby/playlist.xml
+++ b/thirdparty_containers/derby/playlist.xml
@@ -21,6 +21,9 @@
 		<levels>
 			<level>sanity</level>
 		</levels>
+		<subsets>
+			<subset>9+</subset>
+		</subsets>
 		<groups>
 			<group>external</group>
 		</groups>

